src/transformers/models/siglip/convert_siglip_to_hf.py

```diff
@@ -263,35 +278,58 @@ def convert_siglip_checkpoint(model_name, pytorch_dump_folder_path, verify_logit
 
     # get checkpoint
     checkpoint = model_name_to_checkpoint[model_name]
+    if (
+        config.vision_config.image_size == 256
+        and config.text_config.vocab_size == 250000
+        and config.vision_config.patch_size == 16
+    ):
+        siglip_sovit_i18_256 = True
```

Comment:
This can be removed in favor of `if model_name == "siglip-so400m-patch16-256-i18n"`

Reply:
I removed this part and `siglip_sovit_i18_256` checks altogether since I was checking it in only one line